src/invoke.ts

```diff
@@ -0,0 +1,15 @@
+import { promises } from '@api3/operations-utilities/dist/index';
```

Comment:
I've fixed operations-utilities (and published it) so you no longer need the full path. Feel free to give it a go.
```suggestion
import { promises } from '@api3/operations-utilities';
```